src/include/execution/sql/runtime_types.h

```diff
@@ -488,9 +513,101 @@ class EXPORT Decimal {
     return *this;
   }
 
+  /**
+   * Get the underlying value of the decimal
+   * @return This decimal value.
+   */
+  T GetValue() const { return value_; }
+
+  /**
+   * Set the underlying value of the decimal
+   * @param value The value to be set.
+   */
+  void SetValue(T value) { value_ = value; }
+
+  /** This function divides with a given decimal
+   * Note we multiply with the 10^precision of the denominator and then
```

Comment:
Nit: this is all implementation details right? Does the caller need to know any of this? If no, I think the comment should move to the implementation then.